algorithms/ar-binsrt/typescript/bin_sort.ts

```diff
@@ -0,0 +1,29 @@
+function binSort(collection: Array<number>, size:number): void {
```

Comment:
Codacy found an issue: [Function 'binSort' has a complexity of 8. Maximum allowed is 6.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6395193)